test/verifiedTests/analysis/testGeneMCS/testGeneMCS.m

```diff
@@ -0,0 +1,89 @@
+% The COBRAToolbox: testGeneMCS.m
+%
+% Purpose:
+%     - Check the function of geneMCS using different solvers
+%
+% Authors:
+%     - Luis V. Valcarcel 2017-11-17
+%
+
+global CBTDIR
+
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+cd(fileparts(which('testGeneMCS')));
+
+% define the solver packages to be used to run this test
+solverPkgs = {'ibm_cplex','gurobi'}; 
+
+% Load Ecoli core
+load([CBTDIR filesep 'test' filesep 'models' filesep 'mat' filesep 'ecoli_core_model.mat'], 'model');
```

Comment:
Could you please change this to:
`model = getDistributedModel('ecoli_core_model.mat')`
The `getDistributedModel` function allows us to better maintain potential code structure changes (e.g. if the model folder for some reason gets relocated).